### ISSUE TITLE
Bad timing in saved scripts

### ISSUE THREAD

**Issue by eloyesp** [ConradIrwin/showterm#34]
I'm trying to use `showterm` to screencast a cloud-foundry deployment, but for some reason the output is being mixed with the input timing (like if I'm typing the output of the commands) see:

- http://showterm.io/d0f8391e38b45725b1b81
- http://showterm.io/15df6a3f96164c1c14673
- http://showterm.io/a03e06c2069d133ef9ca6

I've tested with different shells (zsh, bash) and with different emulators (xterm, gnome-terminal), have you got any idea?

On the other hand, is this project abandoned or are there any plans to maintain it?

**Comment by eloyesp** [comment]
Well [asciinema](https://asciinema.org/) seems to work fine as an alternative.

**Comment by ConradIrwin** [comment]
I've not seen problems like this before, so something wierd is happening. Are you using something like tmux?

**Comment by eloyesp** [comment]
No, I'm not doing anything like tmux or screen, the application `cf` I'm
calling is written in `go`, so it may be using threads or forks
internally.
